src/mesh/gmsh_utils.jl

```diff
@@ -417,7 +424,13 @@ function gen_rectangle_mesh(
     gmsh.model.mesh.partition(n_partitions)
 
     # Write result
+    rm(output; force = true)
     gmsh.write(output)
+    if write_geo
+        output_geo = output[1:(end - 3)] * "geo_unrolled"
```

Comment:
we could use https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.splitext